### ISSUE TITLE
Error while loading/opening URL

### ISSUE THREAD

**Issue by smseidl** [rob-luke/emails-html-to-pdf#5]
I tried to send a Paypal receipt email to Paperless via the Email to PDF and got the below error.  I don't understand why it's trying to load this page instead of the actual email... thoughts?

```
email2pdf    |
email2pdf    | No attachments in: You have authorized a payment to XXXXXXXX Inc.
email2pdf    |
email2pdf    | PDF: You-have-authorized-a-payment-to-XXXXXXXX_.pdf
email2pdf    |
email2pdf    | PDF: You-have-authorized-a-payment-to-XXXXXXXX_.pdf
email2pdf    | Traceback (most recent call last):
email2pdf    |   File "/app/main.py", line 98, in <module>
email2pdf    |     process_mail(imap_url=server_imap,
email2pdf    |   File "/app/main.py", line 72, in process_mail
email2pdf    |     pdfkit.from_string(html, filename)
email2pdf    |   File "/usr/local/lib/python3.9/site-packages/pdfkit/api.py", line 72, in from_string
email2pdf    |     return r.to_pdf(output_path)
email2pdf    |   File "/usr/local/lib/python3.9/site-packages/pdfkit/pdfkit.py", line 156, in to_pdf
email2pdf    |     raise IOError('wkhtmltopdf reported an error:\n' + stderr)
email2pdf    | OSError: wkhtmltopdf reported an error:
email2pdf    | QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
email2pdf    | Loading page (1/2)
Error: Failed to load https://t.paypal.com/ts?xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx, with network status code 1 and http status code 0 - Connection refused
Printing pages (2/2)                                                        ] 25%
Done                                                                        ]
email2pdf    | Exit with code 1 due to network error: ConnectionRefusedError
email2pdf    |
```

**Comment by rob-luke** [comment]
Whats at the address that its trying to load? Is it a tracker or image etc? But I am with you on this, I'm not sure what the problem might be.

I'll make a PayPal purchase and try and reproduce the issue myself.

**Comment by smseidl** [comment]
Yes, I think in the case for paypal it was a tracker image.  Looks like it
was only 1 px in size.

**Comment by rob-luke** [comment]
Cross referencing #11 and #12

**Comment by rob-luke** [comment]
Closed by #12 
Reopen if not fixed for you
